bundles/org.openhab.binding.nuki/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -191,12 +208,14 @@
 		<category>Door</category>
 		<state readOnly="true">
 			<options>
-				<option value="0">Unavailable</option>
 				<option value="1">Deactivated</option>
 				<option value="2">Closed</option>
 				<option value="3">Open</option>
-				<option value="4">Unknown</option>
+				<option value="4">Door state unknown</option>
 				<option value="5">Calibrating</option>
+				<option value="16">Uncalibrated</option>
+				<option value="240">Removed</option>
+				<option value="255">Unknown</option>
```

Comment:
Please update the file nuki.properties to take into account these new options AND the new configuration parameter.
The easiest is to simply run: `mvn i18n:generate-default-translations` or you can do it manually.

Reply:
What plugin is that? It's not configured on the project, i get 
```
[ERROR] No plugin found for prefix 'i18n' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo]
```
when I run it.

Reply:
As you have not too many changes to do, can you please do the changes manually.
Note that config parameters are generally sorted in alphabetic order.